lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Task/TaskRuntime.cs

```diff
@@ -110,9 +110,9 @@ public Thread StartTaskOnNewThread()
                             "Task running result:\r\n" + System.Text.Encoding.Default.GetString(result));
                     }
                 }
-                catch (TaskStopHandlerException e)
+                catch (TaskStopHandlerException stopException)
```

Comment:
I appreciate the clarification. But this is a pure syntactic change, maybe leave it out of this PR?